lua/nvim_lsp/elmls.lua

```diff
@@ -0,0 +1,44 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+local default_capabilities = lsp.protocol.make_client_capabilities()
+default_capabilities.offsetEncoding = {"utf-8", "utf-16"}
+
+skeleton.elmls = {
+  default_config = {
+    cmd = {"elm-language-server"};
+    filetypes = {"elm"};
+    root_dir = util.root_pattern("elm.json");
+    log_level = lsp.protocol.MessageType.Warning;
+    settings = {};
+    init_options = {
+      elmPath = "elm",
+      elmFormatPath = "elm-format",
+      elmTestPath = "elm-test",
+      elmAnalyseTrigger = "change",
+    };
+    capabilities = default_capabilities;
+    on_init = vim.schedule_wrap(function(client, result)
+      if result.offsetEncoding then
+        client.offset_encoding = result.offsetEncoding
+      end
```

Comment:
nit: `vim.schedule_wrap` isn't needed in this nor the clangd one.

Reply:
This is true. That was an artifact of when I was printing there before.